docs/user/collision-shape-for-physics.rst

```diff
@@ -18,25 +18,17 @@ Use of these is demonstrated in the example content.
    We use a technique called *Fixed Point Iteration* to calculate the water height.
    We gave a talk at GDC about this technique which may be useful to learn more: http://www.huwbowles.com/fpi-gdc-2016/.
 
+The *Visualise Collision Area* debug component is useful for visualising the collision shape for comparison against the render surface.
+It draws debug line crosses in the Scene View around the position of the component.
 
-Compute Shape Queries
----------------------
 
-.. sponsor::
+Collision API Usage
+-------------------
 
-   Sponsoring us will help increase our development bandwidth which could work towards using Burst/Jobs for this feature.
+The collision providers built intout our system perform queries asynchronously; queries are offloaded to the GPU or to spare CPU cores for processing.
+This has a few non-trivial impacts on how the query API must be used.
 
-   .. trello:: https://trello.com/c/qUvB1aSO
```

Comment:
i think this trello could probably be completed as we are technically doing this now - burst job queries.

perhaps we should make a new card for overcoming some of the limtations listed below such as not support depth caches (and perhaps allowing varying water level). that might give the opportunity to retain the sponsorship message.

Reply:
Made this one: https://trello.com/c/EJCQhvsL

LMKWYT

Reply:
that card looks good. im out of time so i cant change the docs but i could probably do it in future work

Reply:
Done.